src/main/java/frc/robot/util/MoTalonFxPID.java

```diff
@@ -16,39 +21,55 @@ public class MoTalonFxPID {
 
     private final Type type;
     private final TalonFX motorController;
+    private final Slot0Configs slotPIDConfigs = new Slot0Configs();
     private double lastReference;
 
     public MoTalonFxPID(Type type, TalonFX controller) {
         this.type = type;
         this.motorController = controller;
+
+        // Load the motor controller's Slot 0 PID values into slotPIDConfigs
+        this.motorController.getConfigurator().refresh(slotPIDConfigs);
     }
 
     public Type getType() {
         return type;
     }
 
+    public Slot0Configs getConfigs() {
+        return slotPIDConfigs;
+    }
+
+    public void applyConfigs() {
+        motorController.getConfigurator().apply(slotPIDConfigs);
+    }
+
     public void setP(double kP) {
-        motorController.config_kP(0, kP);
+        slotPIDConfigs.kP = kP;
+        applyConfigs();
     }
 
     public void setI(double kI) {
-        motorController.config_kI(0, kI);
+        slotPIDConfigs.kI = kI;
+        applyConfigs();
     }
 
     public void setD(double kD) {
-        motorController.config_kD(0, kD);
+        slotPIDConfigs.kD = kD;
+        applyConfigs();
     }
 
     public void setFF(double kFF) {
-        motorController.config_kF(0, kFF);
+        slotPIDConfigs.kV = kFF;
+        applyConfigs();
     }
 
     public void setIZone(double iZone) {
-        motorController.config_IntegralZone(0, iZone);
+        // TODO: This appears to no longer be supported by Phoenix v6
```

Comment:
While this is unfortunate, there's nothing you can do about it. Luckily, we don't currently use the TalonFXs for position control.